xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -357,7 +359,16 @@ static int PlayerControl(const std::vector<std::string>& params)
 
     if (channel)
     {
-      CFileItem playItem(channel);
+      const auto groupMember =
+          CServiceBroker::GetPVRManager().GUIActions()->GetChannelGroupMember(channel);
+      if (!groupMember)
+      {
+        CLog::Log(LOGERROR, "ResumeLiveTv could not obtain channel group member for channel: %s",
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)

Reply:
updated